### ISSUE TITLE
jscpd 3.2.0 requires Node >= 10

### ISSUE THREAD

**Issue by jazzzz** [kucherenko/jscpd#346]
**Describe the bug**

jscpd does not work on Node, contrary to the documentation and the `package.json` file.

**To Reproduce**

1. Run jscpd with Node 8 (I tried with Node 8.16.1)

Error:

```
/home/project/node_modules/fs-extra/lib/mkdirs/make-dir.js:86
      } catch {
              ^

SyntaxError: Unexpected token {
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:617:28)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/project/node_modules/fs-extra/lib/mkdirs/index.js:3:44)
``` 

**Expected behavior**
The doc and the package.json should indicate that Node 10 is required.

**Comment by kucherenko** [comment]
working on it, thank you

**Comment by kucherenko** [comment]
fixed at jscpd@3.2.1

**Comment by jazzzz** [comment]
Great! Thanks
